core/notifications/notifications.go

```diff
@@ -135,3 +224,44 @@ func (n *notificationService) List(limit int64, includeRead bool) ([]*model.Noti
 func (n *notificationService) isNotificationRead(notification *model.Notification) bool {
 	return notification.GetStatus() == model.Notification_Read || notification.GetStatus() == model.Notification_Replied
 }
+
+func (n *notificationService) loadNotificationObject(ctx context.Context) {
+	uk, err := domain.NewUniqueKey(sb.SmartBlockTypeNotificationObject, "")
+	if err != nil {
+		log.Errorf("failed to get notification object unique key: %v", err)
+		return
+	}
+	spc, err := n.spaceService.GetPersonalSpace(ctx)
+	if err != nil {
+		log.Errorf("failed to get personal space for notifications: %v", err)
+		return
+	}
+	n.notificationId, err = spc.DeriveObjectID(ctx, uk)
+	if err != nil {
+		log.Errorf("failed to get notification object: %v", err)
+		return
+	}
+	ctxWithPeer := peer.CtxWithPeerId(ctx, peer.CtxResponsiblePeers)
+	_, err = spc.GetObject(ctxWithPeer, n.notificationId)
+	if err != nil && errors.Is(err, net.ErrUnableToConnect) {
```

Comment:
Are you sure that this is what it will return now? I think it is possible that we will just hang here, because we wait forever for some peers to connect (see `GetResponsiblePeers`) 

This can happen especially if we upgrade from prev version and start the app with no connection to nodes. I guess some long timeout will be working here (like 20 sec maybe?)

Also even in this case it is still possible that we will miss the document and it will load. We need to see that we won't get overloaded with a lot of notifications in case the account is large

Reply:
But again maybe I overestimate the problem tbh

Reply:
I rewrite so now it works like this
1. We get tech space
2. We try to derive notification object in tech space
3. If we error TreeAlreadyExist, we try to derive id

I think we have GetObject() call in `DeriveTreeObject`, probably it should be called with timeout?